plugin/storage/integration/integration.go

```diff
@@ -224,6 +224,40 @@ func (s *StorageIntegration) testGetLargeSpan(t *testing.T) {
 	}
 }
 
+func (s *StorageIntegration) testGetLargeSpanWithDuplicateIDs(t *testing.T) {
```

Comment:
why have a separate test for this, instead of changing the existing one?

Reply:
Hey, just to clarify why I created a separate test—this was based on the earlier discussion in this issue where we agreed that having an explicit test for duplicate span IDs would be useful. It helps document and validate this specific behavior clearly without mixing it with the original large trace test, which is focused on unique IDs.

Reply:
The 10'000 spans test is more about the scale rather than ID uniqueness. The test that allows duplicate IDs is a superset of the other test, so I don't see why we need to run both.

Reply:
Thanks for the feedback! I can merge the two into one test that verifies both scalability and the behavior with duplicate IDs. Would that work?

Reply:
yes, thanks